extension/observer/k8sobserver/handler.go

```diff
@@ -76,6 +78,18 @@ func (h *handler) OnUpdate(oldObjectInterface, newObjectInterface interface{}) {
 			newEndpoints[e.ID] = e
 		}
 
+	case *v1.Service:
+		newService, ok := newObjectInterface.(*v1.Service)
+		if !ok {
```

Comment:
I'm not sure why we're not info/warn-level logging these as programming errors. Does it make sense to begin doing so here?

Reply:
I added warn statements, please have a look